lib/Net/Easypost/Address.pm

```diff
@@ -162,14 +174,19 @@ sub merge {
 =method verify
 
 This method takes a L<Net::Easypost::Address> object and verifies its underlying
-address
+address.
+
+If a non-US address is asked for verification, a warning will be
+emitted and the object itself will be returned.
 
 =cut
 
 sub verify {
     my $self = shift;
-    use Data::Dumper;
-
+    if ($self->country ne 'US') {
```

Comment:
I'm confused, this still doesn't appear to support international shipping. Now it just explicitly states it is not supported. Is that what you intended?